### PR TITLE
Allow listener for native Animated.event

### PR DESCRIPTION
**Pull request by janicduplessis**
We used to not send events to JS when there was a native Animated.event but we do now so we can easily enable the listener property.

**Test plan**
Tested that the listener gets called when using native Animated.event.

### CODE REVIEW

**Comment by janicduplessis** [comment]
cc @ide

File: Libraries/Animated/src/AnimatedImplementation.js [hunk 1]
```diff
@@ -2215,27 +2211,31 @@ class AnimatedEvent {
...
-    return (...args) => {
-      const traverse = (recMapping, recEvt, key) => {
-        if (typeof recEvt === 'number' && recMapping instanceof AnimatedValue) {
-          recMapping.setValue(recEvt);
-        } else if (typeof recMapping === 'object') {
-          for (const mappingKey in recMapping) {
-            traverse(recMapping[mappingKey], recEvt[mappingKey], mappingKey);
+    if (this.__isNative) {
+      return this._listener;
+    } else {
```
Comment (ide) [inline Libraries/Animated/src/AnimatedImplementation.js]: style: remove the else, we've already bailed out of the native case

**Comment by janicduplessis** [comment]
@ide Done!

**Comment by ide** [comment]
@facebook-github-bot shipit

**Comment by janicduplessis** [comment]
@ide Could you try shipping this again

**Comment by ide** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**